pantheon/src/main/java/tech/pegasys/pantheon/cli/DefaultCommandValues.java

```diff
@@ -34,6 +34,7 @@
   String PANTHEON_HOME_PROPERTY_NAME = "pantheon.home";
   String DEFAULT_DATA_DIR_PATH = "./build/data";
   String MANDATORY_INTEGER_FORMAT_HELP = "<INTEGER>";
+  String DURATION_HELP = "<INTEGER>";
```

Comment:
Unless it's a new string simply re-use the existing generic strings.  This can be deleted until we go to duration objects.